CONTRIBUTING.md

```diff
@@ -98,8 +98,8 @@ Not sure what to write about, take a look at our [issues](https://github.com/sel
 
 ## Other Resources
 
-For help with Markdown:
-
-[Markdown -- Github Help](https://help.github.com/en/github/writing-on-github)
-
-[Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+For help with Markdown:  
```

Comment:
Maybe remove this heading?

Reply:
Could comment it out for now. Add it back of a different section of help is added?